src/utils/Exception.js

```diff
@@ -0,0 +1,37 @@
+const { ErrorMessage, StaticValue } = require("../constants/Constants");
+
+const Exception = {
+  checkCarInput(input) {
+    this.checkInputLength(input);
+    this.checkDuplicateInput(input);
+  },
+
+  checkWhiteSpace(input) {
+    return StaticValue.REGEX_WHITESPACE.test(input);
+  },
+
+  checkInputLength(input) {
+    input.forEach((name) => {
+      if (
+        name.length > StaticValue.CAR_NAME_LIMIT ||
+        this.checkWhiteSpace(name)
+      ) {
+        throw new Error(ErrorMessage.NAME_INPUT);
+      }
+    });
+  },
+
+  checkDuplicateInput(input) {
+    if (new Set(input).size !== input.length) {
+      throw new Error(ErrorMessage.NAME_DUPLICATION);
+    }
+  },
+
+  checkMoveCountInput(input) {
+    if (isNaN(input) || Number(input) <= 0 || this.checkWhiteSpace(input)) {
+      throw new Error(ErrorMessage.MOVE_INPUT);
+    }
+  },
```

Comment:
1. input 타입의 값으로 boolean이 들어오게 되면 0, 1로 치환이되고, 이는 isNaN으로 필터링 되지 않습니다.
2. undefined, null 등의 값 또한 NaN이 아니기 때문에 위험해요.

물론 해당 Exception 객체를 호출하는 함수에서 잘 핸들링하면 되겠지만, 가장 원자 단위의 함수에도 방어 코드를 두면 좋을 것 같아요! input의 default value를 0으로 해두거나,  혹은 Number로 변환한 변수를 선언해두고 조건문에 넣으면 어떨까요?

```suggestion
  checkMoveCountInput(input = 0) {
    const parsedInput = Number(input);
    if (isNaN(parsedInput) || parsedInput <= 0 || this.checkWhiteSpace(input)) {
      throw new Error(ErrorMessage.MOVE_INPUT);
    }
  },
```

Reply:
처음부터 input 값을 넘겨줄 때 `parseInt()`한 후 넘겨 주는 것으로 변경했습니다! 전에는 `Controller`에서도 input을 number 타입으로 변경했었는데, 이렇게 해서 반복적인 과정도 줄이고 예외 사항 검증할 때도 공백 검사도 생략하려고 했습니다!